docs/css/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+	font-family: "JetBrains Mono";
+	src: 	url(/fonts/JetBrainsMono-Regular.woff2) format("woff2"),
```

Comment:
Are we allowed to redistribute this font?

Reply:
according to jetbrains: "free and open source" https://www.jetbrains.com/lp/mono/

Reply:
I love this license FAQ. Right to the point https://www.jetbrains.com/lp/mono/#license